storm-client/src/jvm/org/apache/storm/generated/ClusterSummary.java

```diff
@@ -63,8 +60,6 @@ public static _Fields findByThriftId(int fieldId) {
       switch(fieldId) {
         case 1: // SUPERVISORS
           return SUPERVISORS;
-        case 2: // NIMBUS_UPTIME_SECS
-          return NIMBUS_UPTIME_SECS;
         case 3: // TOPOLOGIES
           return TOPOLOGIES;
```

Comment:
Should we keep the sequence number as it is now ?

Reply:
It's my impression that if we don't keep the field ids as they are now, it won't be possible for people to do a rolling upgrade. E.g. if we changed TOPOLOGIES to have id 2, then someone doing a rolling upgrade might have a machine send ClusterSummary with TOPOLOGIES id 3 to an upgraded machine where TOPOLOGIES has id 2, which breaks deserialization.

Reply:
This is generated code. The comment about this is here https://github.com/apache/storm/pull/2761/files#diff-61bbff6c872f2afa10a3b695c1cfe4a3R209